projects/cm_mcu/clocksynth.c

```diff
@@ -113,20 +119,25 @@ static int write_register(int RegList[][2], int n_row)
   int HighByte = -1;
   int status = -10;
   for (int i = 0; i < n_row; i++) {
-    int NewHighByte = RegList[i][0] >> 8; // most significant 8 bits, 16 bits in total
+    uint8_t NewHighByte = RegList[i][0] >> 8; // most significant 8 bits, 16 bits in total
     if (NewHighByte != HighByte) {
       ChangePage = true;
     }
     else {
       ChangePage = false;
     }
     HighByte = NewHighByte;
-    int LowByte = RegList[i][0] - (NewHighByte << 8);
+    uint8_t LowByte = RegList[i][0] - (NewHighByte << 8);
+    
     if (ChangePage) {
+      //      snprintf(m, SCRATCH_SIZE, "ChangePage: 0x%02x 0x01 1 0x%02x\r\n", CLOCK_SYNTH_I2C_ADDRESS, NewHighByte);
```

Comment:
do you want to delete this debugging code (also on line 139)

Reply:
fixed in commit 6258e99